.github/workflows/generate-docs.yaml

```diff
@@ -0,0 +1,54 @@
+name: Generate Documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Generate documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.16'
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
```

Comment:
Is this the path pip actually uses to cache requirements?

Reply:
yes

Reply:
Check it, In my action...i just uploaded 22MB cache https://github.com/evalsocket/flytectl/runs/2753321067?check_suite_focus=true#step:15:3